spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/core/SubscribingByGroupOperation.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.core;
+
+import java.util.function.Consumer;
+
+/**
+ * Operations for subscribing to a destination with a consumer group.
+ *
+ * @author Warren Zhu
+ */
+public interface SubscribingByGroupOperation<D> {
```

Comment:
Same `-ing` format as above.

Reply:
According to the class name,  is there other subscription type? e.g., by non-group operation? Wonder should we define subscription in the same class instead of divide into different classes for group operation and non-group operation.

Reply:
You should have seen `SubscribingOperation` before. Combining will cause some methods throwing unsupported exception. Confusing user.